.coafile

```diff
@@ -6,6 +6,7 @@ bears = SpaceConsistencyBear
 [markdown]
 files = **.md
 bears = MarkdownBear
+max_line_length = 79
```

Comment:
can we set this to 80, and that should mean there are less changes in the PR?

Reply:
We can. But do I have to reset everything? That's gonna be pain.

Reply:
Yes. Objective is minimal change of line attribution

Surya already did change lines using 80 cpl. Annoying to have another large set if changes without good reason.